addons/purchase/report/purchase_report.py

```diff
@@ -27,7 +27,7 @@ class PurchaseReport(models.Model):
     product_id = fields.Many2one('product.product', 'Product', readonly=True)
     partner_id = fields.Many2one('res.partner', 'Vendor', readonly=True)
     date_approve = fields.Datetime('Confirmation Date', readonly=True)
-    product_uom = fields.Many2one('uom.uom', 'Reference Unit of Measure', required=True)
+    product_uom_id = fields.Many2one('uom.uom', 'Reference Unit of Measure', required=True)
```

Comment:
```suggestion
    product_uom_id = fields.Many2one('uom.uom', 'Reference Unit of Measure', readonly=True)
```

Purchase report is a sql view, nobody is expected (nor allowed to) create records of that model, this `required` was probably a mistake.